doc/dftb+/manual/dftbp.tex

```diff
@@ -458,6 +458,27 @@ \subsubsection{Optimiser: FIRE}
 \end{verbatim}
 
 
+\subsubsection{Optimiser: SteepestDescent}
+\label{sec:dftbp.Optimiser.SteepestDescent}
+
+Steepest descent optimiser, that proposes displacements according to the (scaled) gradients.
+
+\begin{ptable}
+  \kw{Scaling} & r &  & 1.0 & \\
+\end{ptable}
+\begin{description}
+\item[\is{Scaling}] Scaling factor of gradients.
+\end{description}
+
+\begin{verbatim}
+Driver = GeometryOptimization {
+  Optimizer = SteepestDescent {
+    Scaling = 1.0
```

Comment:
Possible 'ScalingFactor' is clearer, or the old one uses 'StepSize'. Which of the three do you think is better?

Reply:
From those three options I would probably choose ''ScalingFactor". I will adapt it.

Reply:
Done